### ISSUE TITLE
#-15068342------------- Buttons must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#522]
<strong>Source:</strong>`<button aria-label="" class="icon-end-empty" type="button"><calcite-icon class="icon icon--start" aria-hidden="true" icon="refresh" scale="s" calcite-hydrated=""></calcite-icon></button>`
<strong>Element Location: </strong>[".t-refreshPanelBtn",".icon-end-empty[aria-label=\"\"]"]
<strong>Impact: </strong>critical
<strong>failureSummary:</strong> Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"
Tags: cat.name-role-value,wcag2a,wcag412,section508,section508.22.a,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
